# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gdm/minimal_uid_1000.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

users are >= 1000, not 100. (this is useful when you enable 'browse' on gdm)
Alejandro

--- ./config/gdm.conf.in.orig	2006-01-28 12:34:35.000000000 -0300
+++ ./config/gdm.conf.in	2006-01-28 12:47:17.000000000 -0300
@@ -271,7 +271,7 @@
 # These are things excluded from the face browser, not from logging in
 #Exclude=bin,daemon,adm,lp,sync,shutdown,halt,mail,news,uucp,operator,nobody,gdm,postgres,pvm,rpm,nfsnobody,pcap
 # As an alternative to the above this is the minimum uid to show
-#MinimalUID=100
+MinimalUID=1000
 # If user or user.png exists in this dir it will be used as his picture
 #GlobalFaceDir=@EXPANDED_DATADIR@/faces/
 # File which contains the locale we show to the user.  Likely you want to use
